document_page_approval/models/document_page_history_workflow.py

```diff
@@ -36,7 +39,7 @@ def page_approval_draft(self, cr, uid, ids, context=None):
             'email_template_new_draft_need_approval')[1]
         for page in self.browse(cr, uid, ids, context=context):
             if page.is_parent_approval_required:
-                self.pool.get('email.template').send_mail(
+                self.pool.get('mail.template').send_mail(
```

Comment:
The following element are still using the ald_api interface. That's why i didn't change them
get_object_reference(
            self.env.cr, self.env.uid,
            'document_page_approval',
            'email_template_new_draft_need_approval')[1]
self.pool.get('mail.template').send_mail(
                    self.env.cr, self.env.uid,
                    template_id,
                    page.id,
                    force_send=True
                )
self.pool.get('mail.message')..create(
                self.env.cr, self.env.uid,
                {'res_id': page_history.page_id.id,
                 'model': 'document.page',
                 'subtype_id': subtype_id,
                 'body': _('New version of the document %s'
                           ' approved.') % page_history.page_id.name
                 }
            )